languages/elixir/exercises/concept/exceptions/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+## General
+
+- Read the documentation:
+  - [Elixir Getting Started - Errors][getting-started-errors]
+  - [Elixir documentation - defexception][defexception]
+  - [Elixir documentation - Exception Behaviour][exception-behaviour]
+- Read the code snippets from the introduction
+
+## 1. Error for Division by Zero
+
+- implement the module, specifying the message using `defexception`
```

Comment:
Following the rule that we shouldn't reveal function names in hints, this should say something like:
```suggestion
- implement the module, specifying the message using a special [built-in macro for defining exceptions][defexception]
```